pros/serial/devices/vex/v5_device.py

```diff
@@ -35,8 +36,7 @@ def filter_v5_ports(p, locations, names):
     # Doesn't work on macOS or Jonathan's Dell, so we have a fallback (below)
     user_ports = [p for p in ports if filter_v5_ports(p, ['2'], ['User'])]
     system_ports = [p for p in ports if filter_v5_ports(p, ['0'], ['System', 'Communications'])]
-    joystick_ports = []  # joystick comms are very slow/unusable
-    # joystick_ports = [p for p in ports if filter_v5_ports(p, ['1'], ['Controller'])]
+    joystick_ports = [p for p in ports if filter_v5_ports(p, ['1'], ['Controller'])]
 
     # TODO: test this code path (hard)
```

Comment:
I feel like we were eventually able to test this code path. If that's right, do you mind taking this out while you're making changes to this file? 😃